code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -229,7 +229,7 @@ function highlightTerms(el) {
 							console.log(newWord)
 					}
 					newText += newWord || words[w].replace("<", "&lt;");
-					newText += w >= words.length ? '' : ' ';
+					newText += w >= words.length - 1 ? '' : ' ';
```

Comment:
better yet, remove the -1 and replace >= with >

Reply:
not gonna work, it's the for statement that causes it, look closely, because I don't know how to explain it right